node_monitor/load_config.py

```diff
@@ -18,6 +18,9 @@
 DB_NAME             = os.environ.get('DB_NAME',             '')
 DB_PORT             = os.environ.get('DB_PORT',             '')
 FEEDBACK_FORM_URL   = os.environ.get('FEEDBACK_FORM_URL',   '')
+NODE_MONITOR_URL    = os.environ.get('NODE_MONITOR_URL',    '')
+email_admins_list   = os.environ.get('EMAIL_ADMINS_LIST',   '')
+EMAIL_ADMINS_LIST   = email_admins_list.split(',')
```

Comment:
Add this to `.env.example`

Also, why not just call .split() directly on `os.environ.get('EMAIL_ADMINS_LIST',   '')`?

That `email_admins_list` looks weird.
Also be careful with this. If data gets input wrong, it wont work.